pywinauto/controls/uia_controls.py

```diff
@@ -629,6 +657,26 @@ def get_item(self, row):
     item = get_item  # this is an alias to be consistent with other content elements
 
     #-----------------------------------------------------------
+    def get_item_rect(self, item_index):
+        """Return the bounding rectangle of the list view item
+        
+        The interface is kept mostly for a backward compatibility
+        with the native ListViewWrapper interface"""
```

Comment:
Closing triple double quotes should be on a separate line (landscape minor warning).